docker/Dockerfile.base

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:18.04
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt-get install -y \
+    autoconf \
+    automake \
+    bison \
+    ccache \
+    cmake \
+    ctags \
+    curl \
+    doxygen \
+    flex \
+    fontconfig \
+    gdb \
+    git \
+    gperf \
+    iverilog \
```

Comment:
It seems that this docker is the base for build compatibility checks, you can remove iverilog which is required by regression tests only.

Reply:
great, i'll remove it from the base.

Reply:
It will be better if you just copy .github/workflows/install_dependency.sh in docker image and install. 
That will ensure single point change
```
COPY .github/workflows/install_dependency.sh .
RUN source install_dependency.sh
```